apps/billing/payments/zibal_client.py

```
@@ -145,182 +155,150 @@ def request_payment(
-                'success': validated.result in [self.RESULT_SUCCESS, self.RESULT_ALREADY_PAID],
-                'is_duplicate': validated.result == self.RESULT_ALREADY_PAID
+                'success': validated.result == self.RESULT_SUCCESS,
+                'is_duplicate': validated.result == self.RESULT_ALREADY_PAID,
             }
-            
+
         except ValidationError as e:
-            logger.error("Invalid verify response format", exc_info=True)
-            raise ZibalError(f"Invalid verify response: {str(e)}")
-    
+            logger.error("Invalid Zibal verification response", exc_info=True)
```

Ruff (0.14.0)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)